moc-health-check/test_setup.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import os
+import shlex
+import shutil
+
+"""
+Creates "test" directories of specific size on which the pman and pfioh tests will be performed
+"""
+
+# erases contents of directory in temp
+def clean():
+    if os.path.isdir("/tmp/files/"):
+        shutil.rmtree('/tmp/files/')
+
+# creates a file with specific size in MB
+def create(SIZE, NAME):
+    with open(f'/tmp/files/{SIZE}{NAME}' , 'wb') as bigfile:

```

Comment:
Why put the size in the filename?  Will it ever use more than 1 file?

Reply:
It would create 10 files so each of them must have a unique name

Reply:
Ah I see now, thanks!